src/ol/webgl/Helper.js

```diff
@@ -302,8 +304,6 @@ function releaseCanvas(key) {
  *   ```
  *
  * For an example usage of this class, refer to {@link module:ol/renderer/webgl/PointsLayer~WebGLPointsLayerRenderer}.
- *
- * @api
```

Comment:
Do existing applications continue work with the changes you made to the WebGL helper, e.g. without passing a `frameState` to `useProgram()`? If so, this `@api` annotation and those for the methods can stay for now.

Reply:
Or we keep it like you have it (with the @api annotations removed), and publish v7.0. What do others think?

Reply:
Unfortunately now, this change to the helper is a breaking change, and a necessary one (although it may be worth it to investigate some more)